src/components/Items/Question/Body.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div class="overflow-y-scroll flex flex-col">
+    <!-- question text -->
+    <p class="m-2 sm:m-4 mx-4 md:mx-6 xl:mx-10 font-bold text-lg md:text-xl lg:text-2xl">
+      {{ questionText }}
+    </p>
+    <!-- option container -->
+    <!-- v-model="localSelectedOption" -->
+    <div class="flex mx-4 md:mx-6 xl:mx-10">
+      <ul class="w-full">
+        <li class="list-none">
+          <div
+            v-for="(option, optionIndex) in options"
+            :key="optionIndex"
+            class="p-2 sm:p-4 mb-2 text-lg md:text-xl lg:text-2xl rounded-md"
+            :class="optionBackgroundClass(optionIndex)"
+          >
+            <!-- each option is defined here -->
+            <!-- adding <label> so that touch input is just
+                  not limited to the radio button -->
+            <label class="flex content-center">
+              <!-- understand the meaning of the keys here:
+               https://www.w3schools.com/tags/att_input_type_radio.asp -->
+              <input
+                type="radio"
+                name="questionOptions"
```

Comment:
name is not required here. It would've been required if we were using a form HTML element